tools/test.py

```diff
@@ -938,11 +938,14 @@ def GetV8InspectorEnabledFlag():
   # is currently when Node is configured --without-ssl and the tests should
   # still be runnable but skip any tests that require ssl (which includes the
   # inspector related tests).
-  with open('config.gypi', 'r') as f:
-    s = f.read()
-    config_gypi = ast.literal_eval(s)
-    return config_gypi['variables']['v8_enable_inspector']
-
+  try:
```

Comment:
why not replace with `exec(vm + '-p process.config.variables.v8_enable_inspector')`

Reply:
Not for this PR but I wonder if we should add the inspector to `process.features`.